config/schemas/config.yaml

```
@@ -5997,6 +5997,19 @@ properties:
             title: Network Policies Fluentd Enabled
             type: boolean
             default: true
+          extraOutput:
+            title: Extra Output/Plugin Rule
+            description: |-
+                Configure extra output/plugin egress rules.
```

```suggestion
            title: Extra Output Rule
            description: |-
                Configure extra output egress rules.
```
The output/plugin was depending on change of `extraOutput`. :slightly_smiling_face: